flask/src/main.py

```diff
@@ -191,17 +216,50 @@ def products():
         sentry_sdk.capture_exception(err)
         raise (err)
 
-    try:
-        with sentry_sdk.start_span(op="/api_request", description="function"):
-            headers = parseHeaders(RUBY_CUSTOM_HEADERS, request.headers)
-            r = requests.get(RUBY_BACKEND + "/api", headers=headers)
-            r.raise_for_status()  # returns an HTTPError object if an error has occurred during the process
-    except Exception as err:
-        sentry_sdk.capture_exception(err)
+
+    print("cache_key: " + str(cache_key))
+    get_api_request(cache_key, ruby_delay_time)
 
     return rows
 
 
+def get_api_request(key, delay):
+    start_time = time.time()
+    with sentry_sdk.start_span(op="ruby.api.cache") as span:
+      cached_response = redis_client.get("ruby.api.cache:" + str(key))
+      span.set_data("description", "ruby.api.request")
+      span.set_data("cache.key", key)
```

Comment:
are these standard cache-related attributes that Sentry expects and knows how to interpret or just custom?